scalding-macros/src/main/scala/com/twitter/scalding/macros/impl/ordered_serialization/providers/PrimitiveOrderedBuf.scala

```diff
@@ -76,7 +76,8 @@ object PrimitiveOrderedBuf {
     val bbPutter = newTermName("write" + shortName)
 
     def genBinaryCompare(inputStreamA: TermName, inputStreamB: TermName): Tree =
-      if (Set("Float", "Double", "Character").contains(javaTypeStr)) {
+      //if (Set("Float", "Double", "Character").contains(javaTypeStr)) {
+      if (true) {
```

Comment:
:) I should have changed that. Sorry. That was a quick hack to force that branch (rather than cutting all the code out). Let me see if the unit tests pass here, then I'll fix this.